integ-tests/src/main/java/com/google/testdata/TestDataHelper.java

```diff
@@ -28,12 +28,15 @@ private TestDataHelper() {}
    */
   public static void generateUsers(Connection connection, int userCount) throws SQLException {
     final String sqlDataPath = "src/main/java/com/google/sql/data/users_data.sql";
+    final String password = "test_" + nanoTime();
 
     List<String> sqlQueries = Collections.nCopies(userCount, SqlHelper.getSql(sqlDataPath));
     sqlQueries =
-        sqlQueries.stream().map(e -> e = format(e, nanoTime())).collect(Collectors.toList());
+        sqlQueries.stream().map(e -> e = format(e, nanoTime(), password)).collect(toList());
```

Comment:
While you are here, could you remove the superfluous assignment in the lambda (i.e. drop `e =` before `format`). Also, the re-assignment of sqlQueries is unnecessary. You can make this quasi final:
```
import static com.google.common.base.Suppliers.memoize;
import static com.google.common.collect.ImmutableList.toImmutableList;
...
ImmutableList<String> sqlQueries = Stream.generate(memoize(() -> SqlHelper.getSql(sqlDataPath))).limit(userCount)
    .map(sqlTemplate -> format(sqlTemplate, nanoTime(), password)).collect(toImmutableList());
```

Note that this also uses `ImmutableList` as we prefer immutability whenever possible.

Reply:
I have `com.google.guava:guava-31.0.1-jre</version>` in my repo but I can't resolve `toImmutableList()` and `memoize`, do I need to attach any another guava lib?

Reply:
Done.